code/modules/paperwork/paper.dm

```diff
@@ -302,11 +304,11 @@
 			return
 
 	else if(istype(P, /obj/item/weapon/stamp))
-		if(!in_range(src, usr) && loc != user && !istype(loc, /obj/item/weapon/clipboard) && loc.loc != user && user.get_active_hand() != P)
```

Comment:
Anyway. I was told by coderbus to remove this line. They said it'll never trigger. 
I have doubts about removing code I didn't write myself, so I will gladly re-instate it if asked.